README.md

```diff
@@ -114,10 +119,16 @@ For your convenience, Squiss provides direct access to the AWS SDK's SQS object,
 
 ## Events
 
+### deleted {Message}
+Emitted when a message is confirmed as being successfully deleted from the queue. The `handled` and `delQueued` events will also be fired for deleted messages, but that will come earlier, when the delete function is initially called.
```

Comment:
Just curious since you are calling `handled` immediately when queuing up the message for deletion, but it doesn't actually get deleted until later. Is it possible for anything to fail between that time of calling `delQueued` and the actual `deleted` whereas having already called handled would be an issue if it never actually got deleted?